lib/stripe/object_types.rb

```diff
@@ -9,6 +9,7 @@ def self.object_names_to_classes
       {
         # data structures
         ListObject::OBJECT_NAME => ListObject,
+        SearchResultObject::OBJECT_NAME => SearchResultObject,
```

Comment:
We'll need to do codegen changes for this right?

Reply:
Yep, I'll revert the change from here.